polygon_api/api.py

```diff
@@ -790,6 +924,17 @@ def issue(self):
             self.config.api_url + self.method_name, files=args)
         return Response(json.loads(response.text))
 
+    def issue_raw(self):
+        """Issues request and returns raw response (useful e.g. for files)"""
+
+        args = list(self.args)
+        args.append(('apiKey', self.config.api_key))
+        args.append(('time', str(int(time.time()))))
+        args.append(('apiSig', self.get_api_signature(args, self.config.api_secret)))
+        response = requests.post(
+            self.config.api_url + self.method_name, files=args)
+        return response.text
+
```

Comment:
- We've changed how the arguments are handled to support binary files in #3

I suggest to change according to it and make `issue` to call `issue_raw`

Reply:
Done, now `issue()` refers to `issue_raw()`